docs/overview.rst

```diff
@@ -48,216 +50,298 @@ username and password or using Microsoft Azure *Active Directory* (AD).
 
-Organization settings
----------------------
+The projects are selected by region, so to see projects deployed
+in other regions than the default, use the dropdown menu at the top left. You
+can also create a new project for the organization in the selected region by
+clicking the *Create project* button in the top right.
 
-The organization settings screen lists the current settings for the
-organization:
+The cog icon next to each project takes you to the Project Settings page, in
+the menu simply called Settings, as :ref:`described further below
+<overview-project-settings>`.
 
-.. image:: _assets/img/org-settings.png
+To change your current project selection, click the project name in the list on
+the main Projects page.
 
-From the main panel, you can edit settings or delete the organization.
+.. NOTE::
 
-If you set a notification email and turn on notifications, CrateDB Cloud will
-send relevant operational notifications via email.
+    The menu bar on the left hand side of the Console is divided in two by a
+    line. By design, all menu items below the bar refer to the currently
+    selected project, which is visible at the top left. The next sections of
+    this Documentation are therefore project specific and here referred to as
+    "Project Overview", "Project Settings" etc. For elegance of design,
+    however, the menu bar simply says "Overview", "Settings", and so forth.
+    These each refer to the project you selected on the Projects page.
 
 
-.. _projects:
+.. _overview-projects-overview:
 
-Projects
-========
+Project Overview
+----------------
 
+The Project Overview page, below the divider line, shows you the services
+deployed in the currently selected project. The overview panel shows the name,
+type, and SaaS subscription for billing purposes associated with each service.
 
-.. _project-create:
+.. image:: _assets/img/project-overviewv2.png
 
-Create a project
+Clicking on a cluster service takes you to the Cluster Overview page, which
+bears the name the organization or project admin chose for the cluster. For
+more details on that page, see below.
+
+
+.. _overview-cluster-overview:
+
+Cluster Overview
 ----------------
 
-If you select *Create project* from the :ref:`organization overview
-<org-overview>` screen, you have the option to create a new project by name:
+The Cluster Overview page gives a broad range of relevant data for the cluster
+selected.
 
-.. image:: _assets/img/create-project.png
+.. image:: _assets/img/cluster-overviewv2.png
 
+It gives you the following information:
 
-.. _project-overview:
+* **Cluster name**: The cluster name.
 
-Project overview
-----------------
+.. NOTE::
 
-The project overview screen lists all currently deployed project services along
-with their associated service name, type, and billing:
+    The cluster name, once set, cannot be changed. It makes up part of the
+    unique cluster ID and is therefore permanent until the cluster is deleted.
```

Comment:
The cluster ID is something different. You can refer to it as cluster URL or fqdn (fully qualified domain name).